### PR TITLE
feat: SEC-957 sanitize markdown

### PR DESCRIPTION
**Pull request by jsedlacek**
https://productboard.atlassian.net/browse/SEC-957

### CODE REVIEW

File: src/transformV2.ts [hunk 1]
```diff
@@ -142,8 +143,8 @@ export function transformV2(options: TransformV2Options): TemplateV2Options {
 
   const direction = options.direction || 'ltr';
   const templateOptions: BaseTemplateOptions = {
-    intro: new SafeString(marked(options.intro, { renderer })),
-    outro: new SafeString(marked(options.outro, { renderer })),
+    intro: new SafeString(xss(marked(options.intro, { renderer }))),
+    outro: new SafeString(xss(marked(options.outro, { renderer }))),
     question: options.question.label,
```
Comment (ksvitkovsky) [inline src/transformV2.ts:148]: @jsedlacek do we want to sanitise question as well?

File: src/transformV2.ts [hunk 2]
```diff
@@ -164,7 +165,10 @@ export function transformV2(options: TransformV2Options): TemplateV2Options {
...
   };
 
-  if (options.question.type === 'single-choice' || options.question.type === 'multiple-choice') {
+  if (
+    options.question.type === 'single-choice' ||
+    options.question.type === 'multiple-choice'
+  ) {
     const choices = options.question.choices.map((choice) => {
       return {
         label: choice,
```
Comment (ksvitkovsky) [inline src/transformV2.ts:174]: @jsedlacek same for choices

File: src/transformV2.ts (hunk 1 above)
Comment (jsedlacek) [reply src/transformV2.ts:148]: @ksvitkovsky question is not a SafeString, thus automatically escaped by Handlebars.

File: src/transformV2.ts (hunk 2 above)
Comment (jsedlacek) [reply src/transformV2.ts:174]: @ksvitkovsky same as for question.

**Review by ksvitkovsky — APPROVED** [review]

**Pull request merged**